src/main/java/com/ververica/flink/table/gateway/SqlCommandParser.java

```diff
@@ -277,6 +284,10 @@ private SqlCommandParser() {
 
 		USE_CATALOG,
 
+		CREATE_CATALOG,
+
+		DROP_CATALOG,
```

Comment:
This can be implemented once that https://issues.apache.org/jira/browse/FLINK-17357 is solved